push/src/main/kotlin/com/kohan/push/firebase/FCMService.kt

```diff
@@ -0,0 +1,112 @@
+package com.kohan.push.firebase
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.auth.oauth2.GoogleCredentials
+import com.kohan.push.dto.FCMMessage
+import com.kohan.push.dto.Message
+import com.kohan.push.dto.Notification
+import com.linecorp.armeria.client.WebClient
+import com.linecorp.armeria.common.HttpData
+import com.linecorp.armeria.common.HttpHeaderNames
+import com.linecorp.armeria.common.HttpMethod
+import com.linecorp.armeria.common.HttpRequest
+import com.linecorp.armeria.common.MediaType
+import com.linecorp.armeria.common.RequestHeaders
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import java.io.FileInputStream
+
+@Service
+class FCMService(
+    @Value("\${kohan.push.firebase.messages.send.url}")
+    private val apiUrl: String,
+    @Value("\${kohan.push.firebase.secret.key.path}")
+    private val firebaseConfigPath: String,
+    private val objectMapper: ObjectMapper,
+) {
+    fun sendNotification(
+        token: String? = null,
+        topic: String? = null,
+        title: String,
+        body: String,
+        image: String?,
```

Comment:
I think it would be better to make the parameters of sendNotification into a class and pass them in